geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/Utils.scala

```diff
@@ -118,6 +115,86 @@ object Utils {
     val path = new Path(pathStr)
     fs.delete(path, true)
   }
+
+  object Configurator extends Logging {
+    type ConfigParser = String => Option[Config]
+
+    def getConfig(configArg: String): Config =
+      Seq(parseString, parseFile)
+       .view.map(_(configArg))
+       .find(_.nonEmpty)
```

Comment:
yeah. i think if it wasn't a map and find it would be
```
      Seq(parseString, parseFile)
        .view.flatMap(_(configArg))
        .headOption
        .getOrElse(throw new ParameterException(s"Unable to parse Converter config from argument $configArg"))
```

either works. the view I think makes them fairly equivalent?